packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.html

```diff
@@ -0,0 +1,111 @@
+                            <oui-button
+                                on-click=":: $ctrl.checkInputAndGenerateLink()"
+                                data-variant="primary"
+                            >
+                                <span
+                                    data-translate="telephony_line_softphone_generate_link"
+                                >
+                                </span>
+                            </oui-button>
+                        </div>
+                    </div>
+                </oui-field>
+            </div>
+            <div class="row">
+                <oui-field
+                    class="col-md-12"
+                    label="{{ ::'telephony_line_softphone_record_link' | translate }}"
+                >
+                    <oui-clipboard
+                        data-ng-if="$ctrl.model.recordLink!=='-'"
+                        data-model="$ctrl.model.recordLink"
+                    ></oui-clipboard>
+                    <div
+                        class="oui-input-overlay"
+                        data-ng-if="$ctrl.model.recordLink==='-'"
+                    >
+                        <input
+                            type="text"
+                            class="oui-input"
+                            id="recordLink"
+                            value="-"
+                            disabled
+                            data-model="$ctrl.model.recordLink"
+                        />
+                        <span class="oui-icon oui-icon-copy-normal"></span>
+                    </div>
+                </oui-field>
+            </div>
+            <div class="row">
+                <oui-message data-type="info">
+                    <span
+                        data-translate="telephony_line_softphone_enroll_device_theme_description"
+                    ></span>
+                    <br />
+                    <a href="{{:: $ctrl.softphoneLink}}">
+                        <span
+                            data-translate="telephony_line_softphone_enroll_device_theme_link"
+                        ></span>
+                        <i
+                            class="oui-icon oui-icon-arrow-right align-middle"
+                            aria-hidden="true"
+                        ></i>
+                    </a>
+                </oui-message>
+            </div>
+        </form>
+    </div>
+
+    <div class="text-right">
+        <oui-button
+            size="l"
+            data-disabled="!$ctrl.linkGenerated"
+            data-variant="secondary"
+        >
+            <span data-translate="telephony_line_softphone_button_email_link">
+            </span>
+        </oui-button>
+        <oui-button
+            size="l"
+            data-disabled="!$ctrl.linkGenerated"
+            data-variant="primary"
+        >
+            <span data-translate="telephony_line_softphone_button_save"></span>
+        </oui-button>
```

Comment:
```suggestion
    <div class="text-right">
        <oui-button
            size="l"
            data-disabled="!$ctrl.recordLink"
            data-variant="secondary"
        >
            <span data-translate="telephony_line_softphone_button_email_link">
            </span>
        </oui-button>
        <oui-button
            size="l"
            data-disabled="!$ctrl.recordLink"
            data-variant="primary"
        >
            <span data-translate="telephony_line_softphone_button_save"></span>
        </oui-button>
```

Reply:
this will not work since recordLink has a default value '-'